src/Process/GlobeTileProcessing.js

```diff
@@ -12,6 +12,18 @@ let SSE_SUBDIVISION_THRESHOLD;
 
 const worldToScaledEllipsoid = new THREE.Matrix4();
 
+function _preSSE(view) {
+    const canvasSize = view.mainLoop.gfxEngine.getWindowSize();
+    const hypotenuse = canvasSize.length();
+    const radAngle = view.camera.camera3D.fov * Math.PI / 180;
+
+     // TODO: not correct -> see new preSSE
+    // const HFOV = 2.0 * Math.atan(Math.tan(radAngle * 0.5) / context.camera.ratio);
+    const HYFOV = 2.0 * Math.atan(Math.tan(radAngle * 0.5) * hypotenuse / canvasSize.x);
+
+    preSSE = hypotenuse * (2.0 * Math.tan(HYFOV * 0.5));
```

Comment:
Here `preSSE` is shared between all views, therefore between all globe layers. As it depends on the canvas size, therefore on the renderer, therefore on the view, there is a risk of invalid value (basically if someone uses it before calling `_preSSE()`. I suggest making preSSE a field on `view` instead of a module variable. This way, you don't even need to call `_presSSE` on `computeTileZoomFromDistanceCamera`, `view.preSSE` will always be correct :-)

Reply:
yes, 
preSSE is computed with view and camera. It could return in the fields of the camera (in the case where there will be several camera and one viewer)